app/src/androidTest/java/com/example/sharingang/FakePaymentProvider.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.sharingang
+
+import android.content.Context
+import androidx.fragment.app.Fragment
+import com.example.sharingang.items.Item
+import com.example.sharingang.payment.PaymentProvider
+
+class FakePaymentProvider : PaymentProvider {
+    override fun initialize(fragment: Fragment, context: Context) {
+        // Don't have to initialize anything
+    }
+
+    override suspend fun requestPayment(item: Item): Boolean {
+        if (item.id == null || item.price < 0.01) {
+            return false
+        }
+
+        return paymentStatus.status
```

Comment:
```suggestion
        return if (item.id == null || item.price < 0.01) false else paymentStatus.status
```
It's a bit more concise. Do you think this would work?